mltu/dataProvider.py

```diff
@@ -111,8 +108,6 @@ def transformers(self, transformers: typing.List[Transformer]):
             else:
                 self.logger.warning(f"Transformer {transformer} is not an instance of Transformer.")
 
-        return self._transformers
```

Comment:
why you removed this?

Reply:
While having a setter return a value is possible, its not very useful and not considered a best practice. See below for a more detailed explanation:

https://stackoverflow.com/questions/16615866/python-setter-to-also-return-a-value/16615910#16615910